webapp/components/interop.js

```diff
@@ -258,39 +260,108 @@ class WPTInterop extends QueryBuilder(
       this.handleSearchAutocomplete(e.detail.path);
     };
     this.onLoadingComplete = () => {
-      // passRateMetadata contains the url for the JSON blob of allMetrics;
+      // passRateMetadata contains the url for the JSON blob of precomputedInterop;
       // both fetches need to succeed + parse.
```

Comment:
This comment seems out of date now, the following code only considers one thing.

Reply:
Done.